app/views/public/api/v1/models/_category.json.jbuilder

```diff
@@ -1,5 +1,25 @@
-json.id category.id
 json.name category.name
+json.slug category.slug
 json.locale category.locale
 json.description category.description
 json.position category.position
+
+json.related_categories do
+  if category.related_categories.any?
+    json.array! category.related_categories.each do |related_category|
+      json.partial! partial: 'associated_category', category: related_category
+    end
+  end
+end
+
+if category.parent_category.present?
+  json.parent_category do
+    json.partial! partial: 'associated_category', category: category.parent_category
+  end
+end
+
+if category.linked_category.present?
```

Comment:
we will be taking the change for associated category naming as a separate PR?

Reply:
Yes. There is one more article public APIs PR is open, I am adding the changes in that.